octue/cloud/pub_sub/service.py

```diff
@@ -368,3 +383,22 @@ def send_exception_to_asker(self, topic, timeout=30):
         )
 
         topic.messages_published += 1
+
+    def _parse_question(self, question):
+        """Parse a question in the Google Cloud Pub/Sub or Google Cloud Run format.
+
+        :param dict|Message question:
+        :return (dict, str, bool):
+        """
+        try:
+            # Parse Google Cloud Pub/Sub question format.
```

Comment:
There should probably be a bit more in depth comment about why this try/catch doesn't ack in the exception handler.